### PR TITLE
DLS-MSD-APP-000102 : Logout Component

### PR DESCRIPTION
**Pull request by raymundpaulfontanilla**
**Change(s):**

- import use router
- add logout session

### CODE REVIEW

File: src/app/components/systems/hms/UserProfile.js [hunk 1]
```diff
@@ -34,13 +35,28 @@ const items = [
 ];
 
 const UserProfile = ({ collapsed }) => {
+  const router = useRouter();
+  const handleLogout = () => {
+    sessionStorage.removeItem("userToken");
+    if (!sessionStorage.getItem("userToken")) {
+      router.push("/systems/hms/login");
```
Comment (jcbalang-dev) [inline src/app/components/systems/hms/UserProfile.js:42]: @raymundpaulfontanilla redirect this to HMS login instead of website page

**Review by raymundpaulfontanilla — COMMENTED** [review]
This is good

**Review by raymundpaulfontanilla — COMMENTED** [review]
LGTM

**Review by raymundpaulfontanilla — COMMENTED** [review]
This is ok

**Review by jcbalang-dev — COMMENTED** [review]
This is good

**Pull request merged**